addons/project/static/src/scss/project_widgets.scss

```diff
@@ -74,6 +74,30 @@
             }
         }
     }
+    .o_kanban_update_color_20 {
+        border-left-color: $o-success;
+        &:after {
+            background-color: $o-success;
+        }
+    }
+    .o_kanban_update_color_4 {
+        border-left-color: $o-info;
+        &:after {
+            background-color: $o-info;
+        }
+    }
+    .o_kanban_update_color_2 {
+        border-left-color: $o-warning;
+        &:after {
+            background-color: $o-warning;
+        }
+    }
+    .o_kanban_update_color_23 {
+        border-left-color: $o-danger;
+        &:after {
+            background-color: $o-danger;
+        }
+    }
```

Comment:
I'd have kept the oe_kanban_color_ prefix rather than adding this new one (`o_kanban_update_color`) and I would have define the "colors" 20, 21, 22, 23 like here : https://github.com/odoo/odoo/blob/saas-14.5/addons%2Fproject%2Fstatic%2Fsrc%2Fscss%2Fproject_widgets.scss#L11-L22

So we just have to remove the call to kanban_getcolor here below : `'oe_kanban_color_' + record.color.raw_value`

Reply:
Yes, but I think we can define the oe_kanban_color_21 and oe_kanban_color_22 as fallback, just as its done for the color_bubble in the link hereabove :)

Thanks !